### DISCUSSION TITLE
Some in a while Frigate stops detecting and all recordings are gone

### DISCUSSION THREAD

**Discussion by mirisbowring** [blakeblackshear/frigate#15160] [Ask A Question]
Hi,
after Upgrading to v0.14 (already many months ago), I experience this problemn occassionally. 

So in theory my setup "just works" and i only check the UI if something happend i would like to understand.

But sometimes, i visit the UI and all Recordings are gone. Also, I can see movements / Persons on the camera streams but nothing is in the detection bar on top ob the UI. I have a dedicated HDD just for the detections / recordings but this is also completely empty then.

After Restarting the container everything works as before.
Unfortunately there are no logs that would be relevant to this. 

I am using a coral TPU and an Intel 13th gen for transcoding.

Did anyone else have this problem?

**Comment by hawkeye217** [comment]
We'll need to be able to see your config and any relevant logs in order to help.

**Reply by mirisbowring** [reply]
Going to post the config tomorrow- regarding the logs - we need to wait to the next occurrence since i already restarted the container. 😃
